### PR TITLE
Polish login flow v2

### PR DESCRIPTION
**Pull request by kesselb**
**Master**
master

master

**This PR**
this pr

this pr

### CODE REVIEW

**Review by jancborchardt — APPROVED** [review]
Looks better, thanks! :) There’s still that pesky issue of: Container on update screen is too wide #17556

File: core/templates/loginflowv2/grant.php [hunk 1]
```diff
@@ -42,8 +42,7 @@
 			<input type="hidden" name="requesttoken" value="<?php p($_['requesttoken']) ?>" />
 			<input type="hidden" name="stateToken" value="<?php p($_['stateToken']) ?>" />	
 			<div id="submit-wrapper">
-				<input type="submit" id="submit" class="login primary" title="" value="<?php p($l->t('Grant access')); ?>" />
-				<div class="submit-icon icon-confirm-white"></div>
```
Comment (kesselb) [inline core/templates/loginflowv2/grant.php]: Hmm. https://github.com/nextcloud/server/search?q=submit-icon&unscoped_q=submit-icon `.submit-icon` is used by `grant.js` but this script is only loaded for loginflow v1.

**Comment by kesselb** [comment]
Do you want this fancy loading animation back @jancborchardt ?

Peek 2019-10-26 19-53

**Comment by jancborchardt** [comment]
> Do you want this fancy loading animation back @jancborchardt ?
> 
> Peek 2019-10-26 19-53

Is that shown after tapping the button as feedback? If so then yes, feedback is always good. :)

**Comment by kesselb** [comment]
OK. It was broken for login flow v1 either. Fixed both and rewrote it without jquery. Now we need some js power ;)

**Review by skjnldsv — APPROVED** [review]

**Comment by kesselb** [comment]
`grant.js` is not transpiled by babel :disappointed:  Checked `const, classList, forEach` with Can I use and it should work with: https://browserl.ist/?q=last+2+versions%2C+ie+%3E%3D+11%2C+firefox+ESR%2C+not+dead :see_no_evil:

**Comment by kesselb** [comment]
> So we can merge this? Or should we transpile grant.js?

I think `grant.js` works for the supported browsers.

**Pull request merged**